prdoc/pr_3532.prdoc

```diff
@@ -0,0 +1,12 @@
+# Schema: Polkadot SDK PRDoc Schema (prdoc) v1.0.0
+# See doc at https://raw.githubusercontent.com/paritytech/polkadot-sdk/master/prdoc/schema_user.json
+
+title: Remove deprecated `trait Store`
+
+doc:
+  - audience: Runtime Dev
+    description: |
+      The deprecated `trait Store` feature has been removed from the codebase.
+
+crates:
+  - name: frame-support
```

Comment:
also 'frame' crate is changed because it uses frame-support. 


@liamaharon would any of your new tools come handy here to help devs find all affected crates?

Reply:
got it, prdoc updated